### PR TITLE
RDKBDEV-2687 tableSyncHandler callback to avoid undefined behavior while unregistering a row

### PR DESCRIPTION
**Pull request by alinalabasagemcom**

### CODE REVIEW

**Comment by pradeeptakdas** [comment]
Internal ticket for tracking https://ccp.sys.comcast.net/browse/RDKCOM-5091

File: include/rbus.h [hunk 1]
```diff
@@ -549,7 +566,7 @@ typedef rbusError_t (* rbusEventSubHandler_t)(
  * data element, the rbus library checks for NULL and substitutes a pointer
  * to an error handler function for all unused features
  */
-typedef struct rbusCallbackTable_t
+typedef struct rbusElementCallbackTable_t
```
Comment (karuna2git) [inline include/rbus.h:569]: This `rbusElementCallbackTable_t` struct, can we move it to rbus_element.h

**Review by karuna2git — COMMENTED** [review]
Hi the changes looks very good to me.
I have just one review comment where I wanted to move `rbusElementCallbackTable_t` structure to rbus_element.h

It can be merged just moving this.

**Review by karuna2git — COMMENTED** [review]
Lets move `rbusElementCallbackTable_t` to internal header file as a separate PR. I'll get this change merged.

**Review by karuna2git — APPROVED** [review]

**Pull request merged**